adapters/trustedstack/trustedstack.go

```diff
@@ -0,0 +1,138 @@
+package trustedstack
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"net/url"
+
+	"github.com/prebid/openrtb/v20/openrtb2"
+	"github.com/prebid/prebid-server/v2/adapters"
+	"github.com/prebid/prebid-server/v2/config"
+	"github.com/prebid/prebid-server/v2/errortypes"
+	"github.com/prebid/prebid-server/v2/openrtb_ext"
+)
+
+type adapter struct {
+	endpoint string
+}
+
+func (a *adapter) MakeRequests(request *openrtb2.BidRequest, reqInfo *adapters.ExtraRequestInfo) ([]*adapters.RequestData, []error) {
+	var errs []error
+
+	reqJson, err := json.Marshal(request)
+	if err != nil {
+		errs = append(errs, err)
+		return nil, errs
```

Comment:
Since `MakeRequests` only marshals the ortb request. No need to append marshal error in `errs` and return. 

Could directly return marshal error as done below:

```
reqJson, err := json.Marshal(request)
if err != nil {
	errs = append(errs, err)
	return nil, []error{err}
}
```

Reply:
Done. 

errs = append(errs, err)

Is also not needed now i think.